### PR TITLE
Concierge: Migrate Support Image to Webpack

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

This PR does a few things:

- Removes one of the support images, because a duplicate already exists (and the other is unused)
- Migrates them to Webpack; for details on the reasoning behind this, see #36850 
- Makes some alt text translatable (I think it's supposed to be translatable?) 

#### Testing instructions

There should only be three uses of this image: https://github.com/Automattic/wp-calypso/search?q=%22support.svg%22&unscoped_q=%22support.svg%22

Screen Shot 2021-03-03 at 13 28 08

All of them are in regards to Concierge, but admittedly, I'm unable to get them to appear. I tried playing around a bit with this file to no success: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/upsell-nudge/index.jsx

It would be great if the usages could be checked, but I'm pretty sure the code should be fine. The images should be completely identical. :)

### CODE REVIEW

File: client/my-sites/checkout/upsell-nudge/concierge-quickstart-session/index.jsx [hunk 1]
```diff
@@ -225,8 +230,8 @@ export class ConciergeQuickstartSession extends PureComponent {
 					<div className="concierge-quickstart-session__column-doodle">
 						<img
 							className="concierge-quickstart-session__doodle"
-							alt="Website expert offering a support session"
-							src="/calypso/images/illustrations/support.svg"
+							alt={ translate( 'Website expert offering a support session' ) }
```
Comment (bluefuton) [inline client/my-sites/checkout/upsell-nudge/concierge-quickstart-session/index.jsx:234]: Nice catch 👍

**Comment by bluefuton** [comment]
Thanks @Aurorum! Keen to ship this one 🙂 If you're able to give it a quick rebase, we can 🚢 .

**Comment by Aurorum** [comment]
@bluefuton Super sorry for missing your comment back in May - that was a rough week but I didn't realise that I never revisited this PR. This should be good now, I hope! :)

**Review by bluefuton — APPROVED** [review]
Looks good to me 🚢 

Screen Shot 2021-03-03 at 13 28 08

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5556382

Thank you @Aurorum for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5556382).

**Pull request merged**